React/finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,22 @@
+import isArray from './utils';
+
+export function min(...data) {
+  if (data.length === 0) { 
+    return undefined;
+  } else if (isArray(...data)) {
+    return (Math.min(...data[0]));
+  } 
+  return (Math.min(...data)); 
+}
+
+export function copy(...data) {
+  const args = isArray(...data) ? [...data[0]] : { ...data[0] }
```

Comment:
you can save yourself a line of code by doing... `return isArray(...data) ? [...data[0]] : { ...data[0] }`

Reply:
👍